torch/csrc/jit/python_ir.cpp

```diff
@@ -297,6 +297,12 @@ void initPythonIRBindings(PyObject * module_) {
       ss << n;
       return ss.str();
     })
+    .def("sourceLocation", [](Node & n) {
```

Comment:
`getSourceLocation`, it is confusing to have the name be different across Python and C++